src/main/java/com/amazon/opendistroforelasticsearch/sql/optimizer/subquery/model/SubqueryType.java

```diff
@@ -0,0 +1,34 @@
+/*
+ *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.sql.optimizer.subquery.model;
+
+public enum SubqueryType {
+    /**
+     * SELECT * FROM A WHERE a IN (SELECT * FROM B)
+     */
+    IN(0),
+    UNSUPPORTED(32);
```

Comment:
I am ok with leaving this for now, but logically unsupported does not represent a type, we could have types A, B, C, D, E, where B and E are unsupported.